test-fixtures/1281-local-fs-extra-packages/expected-stdout.txt

```diff
@@ -0,0 +1,9 @@
+Consumer:
+  depends:
+    - ExtraPackage
+  package: consumer
+  path: src/Main.purs
+ExtraPackage:
+  depends: []
+  package: extra-package
+  path: ../extra-package/src/Main.purs
```

Comment:
Before this change the output of `spago graph modules` wouldn't have had this section. Packages outside the workspace root were simply ignored and didn't make it into the graph.